config/initializers/slow_ride.rb

```diff
@@ -0,0 +1,37 @@
+require "redis"
+require "slow_ride"
+
+SlowRide.enable_redis do
+  Redis.new(url: ENV["REDIS_SLOW_RIDE_URL"] || ENV["REDIS_URL"])
+end
+
+# 20% Failure rate
+# 100 checks
+# Reset after it hasn't been checked in 5 minutes
+MY_FEATURE = SlowRide::Redis.new(:my_feature, failure_threshold: 0.2, minimum_checks: 100, max_duration: 5.minutes) do
+  Flipper.disable :my_feature
+  Rails.logger.warn "Flipper my_feature disabled"
+end
```

Comment:
The intent is that our SlowRide features are all defined in this file like this one is. They don't have to be `SCREAMING_SNAKE_CASE`, but that's a common convention in Ruby.